code/modules/antagonists/changeling/powers/transform.dm

```diff
@@ -96,7 +96,6 @@
 /obj/item/changeling
 	name = "flesh"
 	slot_flags = ALL
-	allowed = list(/obj/item/changeling)
```

Comment:
This didn't do anything, it is correctly set on `/obj/item/clothing/suit/changeling`